pages/contact-page/contact.html

```diff
@@ -49,48 +50,124 @@
         </div>
     </nav>
 
-    <!-- Main page content - contact form -->
+    <!-- Main page content - contact form. Philip D.-->
+
+    <!-- Container for the background image.-->
+    <div class="container-fluid contact-page-container">    
 
     <!-- Text and content above the contact form itself -->
-    <div class="container">
-        <br/><br/><br/><br/><br/>
-        <div class="row">
-            <div class="col-lg-1">
-                <h5></h5>
-            </div>
-            <div class="col-lg-8">
-                <h2>Contact page text, potentially some images here. PLACEHOLDER - Philip D.</h2>
+<!--       <div class="container">
+          <br/><br/><br/><br/><br/>
+          <div class="row">
+            <div class="col-lg-12">
+                <h3 class="text-center">We also enjoy speaking to fellow whiskey enthusiasts, there is no ask too large or too small. 
+                  If you would like, please submit your details below and we will be in touch as soon as possible.</h3>
             </div>
-            <div class="col-lg-1">
-                <h5></h5>
-            </div>
-        </div>
-    </div>
-
+          </div>
+      </div> -->
 
-    <!-- The contact form itself - plenty more to do here in terms of alignment, exploring different input types, checking reponsiveness, javascript validation and messaging back to user etc. -->
-    <div class="container">
-        <br/><br/>
+      <div class="container-fluid contact-page-container">
+        <div class="visual-overlay">&nbsp;</div>
         <div class="row">
-            <div class="col-lg-3">
-                <h5></h5>
-            </div>
-            <div class="col-lg-4">
-                <form id="contactForm">
+            <div class="col-lg-12">
+                <section class="jumbotron text-center">
+                  <h1 class="jumbotron-header">"Labhair Linn"</h1>
+                  <hr class="hr-black">
+                  <h5>We love speaking with fellow whiskey enthusiasts, there is no ask too large or too small. 
+                    If you would like, please submit your details below and we will be in touch as soon as possible.</h5>
```

Comment:
<br> tag after 'too small' would allow the whole paragraph to flow vertically down the page nicely thought id mention